js/main.js

```diff
@@ -90,17 +91,35 @@ user.on('change:language', function(md, lang) {
 
 });
 
-//keep user and profile urls synced with the server configuration
+app.serverConfigs = new ServerConfigsCl();
+app.serverConfigs.fetch().done(() => {
+  if (!app.serverConfigs.getActive()) {
+    app.serverConfigs.setActive(
+      app.serverConfigs.serverConfigs.create({
```

Comment:
Should this be app.serverConfig instead of app.serverConfig.serverConfig?

I'm assuming it's just a typo. As-is, this stops the app from working, fixing it seems to correct the issue.